generated/nixnet/nixnet_service.h

```diff
@@ -91,6 +94,9 @@ class NiXnetService final : public NiXnet::Service {
   std::map<std::string, std::int32_t> terminalname_output_map_ { {"PXI_Trig0", 1},{"PXI_Trig1", 2},{"PXI_Trig2", 3},{"PXI_Trig3", 4},{"PXI_Trig4", 5},{"PXI_Trig5", 6},{"PXI_Trig6", 7},{"PXI_Trig7", 8},{"FrontPanel0", 9},{"FrontPanel1", 10},{"PXI_Star", 11},{"PXI_Clk10", 12},{"10MHzTimebase", 13},{"1MHzTimebase", 14},{"MasterTimebase", 15},{"CommTrigger", 16},{"StartTrigger", 17},{"FlexRayStartCycle", 18},{"FlexRayMacrotick", 19},{"LogTrigger", 20},{"TimeTrigger", 21},{"NetworkTimePPS", 22},{"NetworkTime1MHz", 23}, };
+  std::map<std::uint32_t, Type> dbproperty_type_map_ { {100663298, ref_array_},{50331649, string_},{33554435, boolean_},{151060481, u64_},{65651, u32_},{65537, u32_},{151060497, u64_},{65553, u32_},{65652, u32_},{65552, u32_},{50397192, string_},{65545, u32_},{83951618, ref_},{100728835, ref_array_},{65568, u32_},{65650, u32_},{65569, u32_},{65570, u32_},{65571, u32_},{65572, u32_},{65573, u32_},{65574, u32_},{65575, u32_},{65579, u32_},{65578, u32_},{65576, u32_},{65577, u32_},{16842800, f64_},{65585, u32_},{65586, u32_},{65588, u32_},{65587, u32_},{65590, u32_},{65591, u32_},{65589, u32_},{65592, u32_},{65593, u32_},{65600, u32_},{65601, u32_},{65602, u32_},{65603, u32_},{65605, u32_},{65606, u32_},{65607, u32_},{65608, u32_},{65609, u32_},{33620053, boolean_},{65616, u32_},{65617, u32_},{65618, u32_},{65619, u32_},{65620, u32_},{100728836, ref_array_},{100728944, ref_array_},{16842865, f64_},{50397189, string_},{100728840, ref_array_},{33619978, boolean_},{65542, u32_},{100728839, ref_array_},{131172, u32_},{33685520, boolean_},{131174, u32_},{131089, u32_},{16908306, f64_},{84017153, ref_},{50462722, string_},{131081, u32_},{167903237, u8_array_},{131104, u32_},{131105, u32_},{131106, u32_},{134348850, u32_array_},{33685552, boolean_},{134348849, u32_array_},{33685539, boolean_},{33685540, boolean_},{33685541, boolean_},{131110, u32_},{131075, u32_},{131152, u32_},{84017217, ref_},{33685568, boolean_},{100794434, ref_array_},{100794435, ref_array_},{50462724, string_},{134348897, u32_array_},{134348899, u32_array_},{100794464, ref_array_},{131079, u32_},{100794376, ref_array_},{33685605, boolean_},{196609, u32_},{50528258, string_},{196617, u32_},{196611, u32_},{16973828, f64_},{84082693, ref_},{16973830, f64_},{16973831, f64_},{33751088, boolean_},{33751089, boolean_},{196658, u32_},{84082739, ref_},{50528264, string_},{50528272, string_},{196626, u32_},{84082705, ref_},{16973843, f64_},{16973844, f64_},{196629, u32_},{50528278, string_},{262153, u32_},{100925441, ref_array_},{84148226, ref_},{262147, u32_},{50593796, string_},{50593799, string_},{84148229, ref_},{84213761, ref_},{50659333, string_},{327689, u32_},{33882128, boolean_},{327700, u32_},{84213777, ref_},{327699, u32_},{327698, u32_},{100990979, ref_array_},{100990980, ref_array_},{327715, u32_},{327717, u32_},{327714, u32_},{33882144, boolean_},{17104934, f64_},{327713, u32_},{17104935, f64_},{327716, u32_},{50659330, string_},{151322665, u64_},{327720, u32_}, };
```

Comment:
Did you already discuss "properties we don't know about" with Harsha and/or Xnet? Are we OK with not exposing a way to access them?